charts/rstudio-pm/values.yaml

```diff
@@ -232,7 +232,10 @@ serviceMonitor:
 # -- config is a nested map of maps that generates the rstudio-pm.gcfg file
 config:
   Server:
-    RVersion: "/opt/R/3.6.2/"
```

Comment:
Since these are the product defaults I feel like removing them entirely would be appropriate. I understand having them explicit for the transition away from RVersion though so perhaps we leave them explicitly set to blank for a while then remove entirely to keep the values clean?

Reply:
Good point on removing them entirely. 

I think we should just remove it entirely in this PR. The change is captured in the changelog, so we don't need to also document the change in the values.yaml.

I will make the update.